compton-conf/compton-conf_fr.ts

```diff
@@ -0,0 +1,190 @@
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="99"/>
+        <source>Don&apos;t draw shadows on DND windows</source>
+        <translation>Ne pas dessiner des ombres sur les fenêtres DND</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="106"/>
+        <source>Avoid drawing shadows on dock/panel windows</source>
+        <translation>Éviter de tirer des ombres sur les fenêtres Dock ou tableau de bord</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="113"/>
+        <source>Enabled client-side shadows on windows</source>
+        <translation>Activer les ombres du côté client des fenêtres</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="120"/>
+        <source>Avoid drawing shadow on all shaped windows</source>
+        <translation>Éviter de tirer ombre sur toutes les fenêtres formées</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="138"/>
+        <source>Opacity</source>
+        <translation>Opacité</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="147"/>
+        <source>The opacity for menus</source>
+        <translation>Opacité pour les menus</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="154"/>
+        <source>Default opacity of inactive windows</source>
+        <translation>Opacité par défaut des fenêtres inactives</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="161"/>
+        <source>Default opacity for active windows</source>
+        <translation>Opacité par défaut des fenêtres actives</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="168"/>
+        <source>Opacity of window titlebars and borders</source>
+        <translation>Opacité des barres de titre et des bordures</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="175"/>
+        <source>XRender backend: Step size for alpha pictures
+(Increasing it may result in less X resource usage)</source>
+        <translation>XRender backend: Taille des étapes pour les images alpha
+(Les augmenter peut entraîner une moindre utilisation des ressources X)</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="183"/>
+        <source>Dim inactive windows</source>
+        <translation>Assombrir les fenêtres inactives</translation>
+    </message>
+    <message>
+        <location filename="../../maindialog.ui" line="190"/>
+        <source>Do not let dimness adjust based on window opacity</source>
+        <translation>Ne pas laisse la luminosité se régler en fonction de l&apos;opacité de la fenêtre</translation>
```

Comment:
I corrected the mistakes, but not the last of pcmanfm-qt. I have not been able to locate it.
Regarding the word 'focus', it is actually used in French, particularly in computer (according to wikipedia)